src/dmd/cli.d

```diff
@@ -711,6 +711,8 @@ dmd -cov -unittest myprog.d
             "list all variables going into thread local storage"),
         Feature("vmarkdown", "vmarkdown",
             "list instances of Markdown replacements in Ddoc"),
+        Feature("safe", "showSystem",
+            "list un-@safe functions not marked @system"),
```

Comment:
```suggestion
            "list `@system` functions which would become `@safe` with `-preview=safedefault`"),
```

Or something similar would be IMO more appropriate. Also I know there's a lot of discussion happening about "unsafe" vs "system", but let's stick to the established precedent here.

Reply:
Done, allthough I emphasized that it will report functions that won't compile with "`@safe by default`".

Reporting all unannotated could be implementet as well.